aptos-move/block-executor/src/executor.rs

```diff
@@ -153,9 +153,7 @@ where
             }
 
             for (id, change) in output.delayed_field_change_set().into_iter() {
-                // if !prev_modified_aggregators.remove(&id) {
-                //     updates_outside = true;
-                // }
+                prev_modified_delayed_fields.remove(&id);
```

Comment:
Earlier, the `updates_outside` flag was set to true when a new aggregator is changed during this incarnation. But now, we are not setting updates_outside flag. What's the rationale?

Reply:
I think there is also a todo below for that

Reply:
yes there is TODO - but I've discussed with @gelash - I don't think we want to update it. It marks later transactions for re-validation - but we don't look at delayed fields during re-valiadation - only before commit. (at the moment)